source/guides/clear/clear-linux-dissector.rst

```diff
@@ -0,0 +1,302 @@
+.. _clear-linux-dissector:
+
+Clear Linux\* Dissector
+#######################
+
+The Clear Linux\* Dissector tool compares contents of Linux-based images. It
+supports the comparison of |CL| and Yocto Project\* images.
+
+.. contents:: :local:
+   :depth: 1
+
+Description
+***********
+
+Clear Linux Dissector provides tools for comparing |CL| images and Yocto Project
+images.
+
+* **View distribution data.**
+
+  View the distribution data of a specific |CL| and Yocto Project distribution
+  side-by-side. Helpful to determine if a |CL| version contains the packages and
+  patches needed to move from a specific Yocto Project distribution to |CL|.
+
+* **Compare images.**
```

Comment:
@arbrindle - Please check/confirm if this tool compares only  CL and Yocto Project images. Also check if this tool will compare a custom CL image (eg made from mixer) to a CL release.

Reply:
This can compare CL to YP and YP to YP; it can't compare a custom CL image

Reply:
Note: it does have the ability to handle a single custom image as part of a "derivative" Clear Linux distribution, but that idea of a derivative is based upon one specific derivative and I'm not sure how applicable that is to any others, so perhaps that's not worth going into.